### PR TITLE
fix disconnection of worker (probably) caused by gevent

### PR DESCRIPTION
**Pull request by xlqian**

### CODE REVIEW

**Comment by kinnou02** [comment]
Can you update the deployement too plz

File: Procfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
-web: ./manage.py runserver
+web: USE_GEVENT=true ./manage.py runserver
 load_realtime: ./manage.py load_realtime
```
Comment (kinnou02) [inline Procfile]: We will need gevent on this one too I think

File: Procfile [hunk 2]
```diff
@@ -1,5 +1,5 @@
-web: ./manage.py runserver
+web: USE_GEVENT=true ./manage.py runserver
```
Comment (kinnou02) [inline Procfile]: Also, you should prefix it with `KIRIN_`

File: kirin/core/handler.py [hunk 3]
```diff
@@ -50,7 +50,7 @@ def persist(real_time_update):
 
 def log_stu_modif(trip_update, stu, string_additional_info):
     logger = logging.getLogger(__name__)
-    logger.info("TripUpdate {vj_id} on {date}, StopTimeUpdate {order} modified: {add_info}".format(
+    logger.debug("TripUpdate {vj_id} on {date}, StopTimeUpdate {order} modified: {add_info}".format(
```
Comment (pbougue) [inline kirin/core/handler.py]: 👍

File: kirin/gtfs_rt/model_maker.py [hunk 4]
```diff
@@ -86,7 +86,7 @@ def build(self, rt_update, data):
 
         The TripUpdates are not yet associated with the RealTimeUpdate
         """
-        self.log.info("proto = {}".format(data))
+        self.log.debug("proto = {}".format(data))
```
Comment (pbougue) [inline kirin/gtfs_rt/model_maker.py]: 👍

**Comment by xlqian** [comment]
@kinnou02 here it is: https://github.com/CanalTP/fab_kirin/pull/15

File: Procfile [hunk 5]
```diff
@@ -1,5 +1,5 @@
-web: ./manage.py runserver
-load_realtime: ./manage.py load_realtime
-worker: celery worker -P gevent -A kirin.tasks.celery
+web: KIRIN_USE_GEVENT=true ./manage.py runserver
+load_realtime: KIRIN_USE_GEVENT=true ./manage.py load_realtime
+worker: celery worker -A kirin.tasks.celery -c 3
```
Comment (pbougue) [inline Procfile]: To be cautious, this should be `-c 1`:
* it's none of this PR's responsibility to deal with that
* bigger problem: we're not sure how kirin will react with 2 poller polling the same thing at the same time (which might happen if poll is slower than beat)

**Pull request merged**